### PR TITLE
Task/header

### PR DESCRIPTION
**Pull request by AnatoliRub**
Implement header for youtube client app

### CODE REVIEW

File: youtube-client/src/app/components/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss'],
+})
+export class LoginComponent {}
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/app/components/login/login.component.ts]: bb2e95c0ad64e0cf7e11c3784685b897681d7b5b

File: youtube-client/src/index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js"></script>
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/index.html]: 76949b8346785a4a7fae0cfcf0b6696317276ba4

File: youtube-client/src/app/components/logo/logo.component.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 0.625rem;
+  transition: 1s;
+
+  &__link {
+    height: 3.125rem;
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/app/components/logo/logo.component.scss]: 185e259653a03b0acf8490b147bbb60e45b1489a

File: youtube-client/src/app/components/svgs/youtube/youtube.component.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-youtube',
+  templateUrl: './youtube.component.html',
+  styleUrls: ['./youtube.component.scss'],
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/app/components/svgs/youtube/youtube.component.ts]: 99080a514898e8c4ec3b8d51177d269f512cc1f4

File: youtube-client/src/app/components/login/login.component.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+    .fa-user-circle {
+      color: $white;
+      font-size: 1.2rem;
+    }
+  }
+}
+
+@media (max-width: 1200px) {
+  .login__account {
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/app/components/login/login.component.scss]: 5a3b4d2345fcbb157784449450c4f2956efa2a9c

File: youtube-client/src/app/app.module.ts [hunk 6]
```diff
@@ -3,9 +3,21 @@ import { BrowserModule } from '@angular/platform-browser';
...
+  declarations: [
+    AppComponent,
+    HeaderComponent,
+    LogoComponent,
+    YoutubeComponent,
+    SearchComponent,
+    LoginComponent,
+  ],
   imports: [BrowserModule, AppRoutingModule],
   providers: [],
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/app/app.module.ts]: cde2585c27a908a5b5ffaa769a741672ce257ef1

File: youtube-client/src/styles.scss [hunk 7]
```diff
@@ -1 +1,76 @@
...
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+/* Variables */
+
+/* Colors */
+$main-backgr: #e5e5e5;
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/styles.scss]: e44b26bbdf9b4ec960e0eb8e640568f6a9601506

File: youtube-client/src/styles.scss [hunk 8]
```diff
@@ -1 +1,76 @@
...
+}
+
+* {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+/* Variables */
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/styles.scss]: ea12c290ccbad48ffd7f6c6d9a752b283c7f4c91

File: youtube-client/src/styles.scss [hunk 9]
```diff
@@ -1 +1,76 @@
...
+$green: #27ae60;
+$red: #eb5757;
+$gray: #efefef;
+$gray-light: #e0e0e0;
+$gray-middle: #c4c4c4;
+$gray-dark: #b1b1b1;
+$black: #4f4f4f;
+$white: #ffffff;
+
+/* Mixins */
```
[the remark this answers is not in the document]
Comment (AnatoliRub) [reply youtube-client/src/styles.scss]: ea12c290ccbad48ffd7f6c6d9a752b283c7f4c91

**Review by StanevichIvan — APPROVED** [review]

**Pull request merged**